kore/src/Kore/Step/EqualityPattern.hs

```diff
@@ -69,7 +72,7 @@ data EqualityPattern variable = EqualityPattern
     , left  :: !(TermLike.TermLike variable)
     , right :: !(TermLike.TermLike variable)
     , ensures :: !(Predicate variable)
-    , attributes :: !Attribute.Axiom
+    , attributes :: !(Attribute.Axiom SymbolOrAlias)
```

Comment:
At this point, the attributes should be verified, so let's use `Attribute.Axiom Symbol` here.